huxley/settings/logging.py

```diff
@@ -15,6 +15,16 @@
             'class': 'logging.StreamHandler',
             'formatter': 'simple',
         },
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': 'huxley/logs/error.log',
```

Comment:
If this handler is for levels DEBUG and above, I don't think the filename should be named `error`.